package.json

```diff
@@ -42,13 +42,13 @@
     "passport": "^0.3.2",
     "passport-helsinki": "City-of-Helsinki/passport-helsinki#fec48d78e0258aa1472369014ffc01cd98c7cdc6",
     "progress": "^1.1.8",
-    "query-string": "^3.0.0",
-    "react": "^0.14.1",
-    "react-bootstrap": "^0.28.1",
-    "react-dom": "^0.14.1",
+    "query-string": "^4.2.2",
+    "react": "^15.1.0",
+    "react-bootstrap": "^0.29.5",
+    "react-dom": "^15.1.0",
     "react-helmet": "^2.2.0",
     "react-intl": "2.0.0-rc-1",
-    "react-leaflet": "^0.10.1",
+    "react-leaflet": "^0.12.0",
```

Comment:
This might need to be `^0.11.5` because of `react-leaflet-draw`. Checking it out right now.